plugins/native/module/vtkQuakeMDLImporter.h

```diff
@@ -0,0 +1,133 @@
+/**
+ * @class   vtkQuakeMDLImporter
+ * @brief   VTK Importer for Quake 1 models in binary .mdl file format
+ */
+
+#ifndef vtkQuakeMDLImporter_h
+#define vtkQuakeMDLImporter_h
+#include <vtkImporter.h>
+
+class vtkQuakeMDLImporter : public vtkImporter
+{
+public:
+  static vtkQuakeMDLImporter* New();
+  vtkTypeMacro(vtkQuakeMDLImporter, vtkImporter);
+
+  void PrintSelf(ostream& os, vtkIndent indent) override;
+
+  ///@{
+  /**
+   * Set/Get the file name.
+   */
+  vtkSetMacro(FileName, std::string);
+  vtkGetMacro(FileName, std::string);
+  ///@}
+
+  /**
+   * Update actors at the given time value.
+   */
+
+  void UpdateTimeStep(double timeValue) override;
+
+  /**
+   * Get the number of available animations.
+   */
+  vtkIdType GetNumberOfAnimations() override;
+
+  /**
+   * Return the name of the animation.
+   */
+  std::string GetAnimationName(vtkIdType animationIndex) override;
+
+  ///@{
+  /**
+   * Enable/Disable/Get the status of specific animations
+   * Only one single animation can be enabled
```

Comment:
this is a limitation of the assimp importer, but can it be implemented correctly here ?